fairscale/utils/parallel.py

```diff
@@ -25,7 +25,7 @@ def chunk_and_pad(tensor: torch.Tensor, num_chunks: int) -> List[torch.Tensor]:
     return chunks
 
 
-def validate_process_group(device: torch.device, process_group: ProcessGroup) -> None:
+def validate_process_group(device: torch.device, process_group: Dict[str, ProcessGroup]) -> None:
```

Comment:
similar to the comment below, I think we can make this function not aware of the specifics of all_gather and reduce_scatter.